indico/core/db/sqlalchemy/protection.py

```diff
@@ -462,10 +462,14 @@ def update_principal(self, principal, read_access=None, full_access=None, permis
                                        old_data=old_data, quiet=quiet)
         return entry
 
-    def get_manager_list(self, recursive=False):
-        managers = {x.principal for x in self.acl_entries if x.has_management_permission()}
+    def get_manager_list(self, recursive=False, include_groups=True):
```

Comment:
For the sake of consistency I would update the signature of `SubContribution.get_manager_list` as well.